doc/pages/faces.asciidoc

```diff
@@ -2,183 +2,188 @@
+fg_color, bg_color, underline_color::
+    A color whose value can be:
+        A named color:::
+            *black*, *red*, *green*, *yellow*, *blue*, *magenta*, *cyan*,
+            *white* *bright-black*, *bright-red*, *bright-green*,
+            *bright-yellow* *bright-blue*, *bright-magenta*, *bright-cyan*,
+            *bright-white*
+       The existing color:::
+            *default*
+        A hexadecimal value:::
+            *rgb:RRGGBB*, *rgba:RRGGBBAA*
+    If left unspecified, the value of *default* is used.
+    Alpha values are used to blend the face onto either its base (see below) or
+    else onto whatever color happens to be used at the moment. For technical
+    reasons, alpha values must be >16.
+
+attributes::
+    A string whose individual letters each set an attribute:
         *u*:::
-            underline
+            Underline
         *c*:::
-            curly underline, takes precedence over underline if both are
-            specified
+            Curly underline
+            Note: This takes precedence over underline if both are specified.
         *r*:::
-            reverse
+            Reverse
         *b*:::
-            bold
+            Bold
         *B*:::
-            blink
+            Blink
         *d*:::
-            dim
+            Dim
         *i*:::
-            italic
+            Italic
         *s*:::
-            strikethrough
+            Strikethrough
         *F*:::
-            final, override the previous face instead of merging with it,
-            and this face will only be replaced if another face with
-            the final attribute is applied
+            Final::::
+                Override the previous face instead of merging with
+                it. Can only be replaced if another face with the final
+                attribute is applied.
         *f*:::
-            final foreground, as final but only applies to face's
-            foreground color
+            Final foreground::::
+                Same as final, but only applies to a face's foreground color.
         *g*:::
-            final background, as final but only applies to face's
-            background color
+            Final background::::
+                Same as final, but only applies to a face's background color.
         *a*:::
-            final attributes, as final but only applies to face's
-            attributes
+            Final attributes::::
+                Same as final, but only applies to a face's attributes.
 
-'base'::
-    The base face on which this face applies, which can be any face name,
-    as long as no cycle is introduced. A face can reference itself, in
-    which case it will apply on top of the parent scope version.
+base::
+    The face onto which the rest of the face applies. Its value can be any
+    face name, as long as no cycle is introduced. A face can reference itself,
+    in which case it will apply on top of the parent scope.
```

Comment:
`parent scope` sounds weird because a face can't apply to a scope.
The old phrasing is `parent scope version` which is better - it means the base is the face of the same name but in the parent scope.

Reply:
Sorry, I don't know what to make of this. The word **version** seems pleonastic to me:

If I have two books called "Moby Dick", each being different in some way (published year, for example), I can refer to one of them by saying either: a) The older one is in better shape, or b) The older version is in better shape.

The word **version** doesn't add anything meaningful. It's a noun redundantly affixed to another noun. The object is the same regardless. Consider, "Roses are pretty" vs "Rose flowers are pretty".

Reply:
To me 

	A face can reference itself, in which case it will apply on top of the parent scope.

sounds a bit like

	Do you want to read this copy of "Moby Dick" or the public library?

From that perspective, adding "version" at the end makes a difference.

I also see your point that it's obvious to humans but it trips up my internal type checker.
Maybe we can find an elegant rephrasing.

Reply:
I doubt that it's obvious to even 1% of native English speakers, let alone humans.

Anyhow, as I was just reverting the wording to "parent scope version", I realized from your comments that what we may want to say is, "version of the parent scope" or "parent scope's version". The preposition in the former, and the possessive in the latter, do much in the way of disambiguation. Would one of these serve us well? Or shall I revert to the original "parent scope version", thus betraying my own linguistic integrity (joking).

Reply:
Good idea, I think both "version of the parent scope" and "parent scope's version" are better.

Reply:
Done